.drone.yml

```diff
@@ -52,45 +52,26 @@ steps:
    - push
    - pull_request
 
-#- name: Unit Tests and SonarQube Scanner
-#  pull: if-not-exists
-#  image: quay.io/ukhomeofficedigital/sonar-scanner-node:9fed2b90853321ac9a9c3da548a180f22c5dc4cb
-#  commands:
-#  - export NOTIFY_API_KEY=dummy
-#  - cd src
-#  - npm install
-#  - npm run test-drone
-#  - cd ..
-#  - sonar-scanner -Dsonar.projectVersion=$${DRONE_BUILD_NUMBER}
-#  settings:
-#    group: build-stage
-#  when:
-#    event:
-#    - push
-#    - pull_request
-
-- name: scan-newly-built-image
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/anchore-submission:latest
+- name: scan-image
   pull: always
-  failure: ignore
-  environment:
+  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
+  resources: 
+    limits:
+      cpu: 1000
+      memory: 1024Mi
+  environment: 
     IMAGE_NAME: quay.io/ukhomeofficedigital/egar-public-site-ui:${DRONE_COMMIT}
     TOLERATE: high
```

Comment:
Is the `TOLERATE` environment variable used in Trivy?
* From the docs, it does not seem to be the case (https://docs.acp.homeoffice.gov.uk/how-to/security/trivy/).